readme.txt

```diff
@@ -1,13 +1,13 @@
-=== Two-Factor ===
-Contributors: georgestephanis, valendesigns, stevenkword, extendwings, sgrant, aaroncampbell, johnbillion, stevegrunwell, netweb, kasparsd
-Tags: two factor, two step, authentication, login, totp, fido u2f, u2f, email, backup codes, 2fa, yubikey
-Requires at least: 4.3
-Tested up to: 5.1
-Stable tag: trunk
-
-Enable Two-Factor Authentication using time-based one-time passwords (OTP, Google Authenticator), Universal 2nd Factor (FIDO U2F, YubiKey), email and backup verification codes.
-
-== Description ==
+=== Two-Factor ===
+Contributors: georgestephanis, valendesigns, stevenkword, extendwings, sgrant, aaroncampbell, johnbillion, stevegrunwell, netweb, kasparsd, alihusnainarshad
+Tags: two factor, two step, authentication, login, totp, fido u2f, u2f, email, backup codes, 2fa, yubikey
+Requires at least: 4.3

```

Comment:
I guess the diff is so large for this file because we're changing the line endings. Could you please ensure we're using `crlf` line endings per:

https://github.com/georgestephanis/two-factor/blob/63de5fc4056308930fad0752e432606a0fbbff6a/.editorconfig#L14-L15